skops/io/tests/test_persist.py

```diff
@@ -1092,3 +1092,10 @@ def test_defaultdict():
     obj_loaded = loads(dumps(obj))
     assert obj_loaded == obj
     assert obj_loaded.default_factory == obj.default_factory
+
+
+@pytest.mark.parametrize("cls", [dict, OrderedDict])
+def test_dictionary(cls):
+    obj = cls({1: 5, 6: 3, 2: 4})
+    loaded_obj = loads(dumps(obj))
+    assert obj == loaded_obj
```

Comment:
Should the type also be checked? E.g. this passes:

```python
assert {2: 20, 1: 10} == OrderedDict([(2, 20), (1, 10)])
assert {1: 10, 2: 20} == OrderedDict([(2, 20), (1, 10)])
```

Reply:
ping @adrinjalali

Reply:
oops, sorry. Added now @BenjaminBossan